src/main.js

```diff
@@ -17,15 +17,60 @@ function buildNewCover() {
   return new Cover(coverImgSrc, title, descriptor1, descriptor2);
 }
 
+// Global cover variables
 var coverImage = document.querySelector('.cover-image');
 var coverTitle = document.querySelector('.cover-title');
 var firstDescriptor = document.querySelector('.tagline-1');
 var secondDescriptor = document.querySelector('.tagline-2');
 var button = document.querySelector('.random-cover-button');
 
+// Global view variables
+var homeView = document.querySelector('.main-cover');
+var savedView = document.querySelector('.saved-covers-section');
+var formView = document.querySelector('.form-view');
 
-var button = document.querySelector('.random-cover-button');
-button.addEventListener('click', updateCover);
+// global control variables
+var homeButton = document.querySelector('.home-button');
+var randomButton = document.querySelector('.random-cover-button');
+var saveButton = document.querySelector('.save-cover-button');
+var formButton = document.querySelector('.make-new-button');
+var viewSavedButton = document.querySelector('.view-saved-button');
+
+// event listeners
+randomButton.addEventListener('click', updateCover);
+formButton.addEventListener('click', switchToFormView);
+viewSavedButton.addEventListener('click', switchToSavedView);
+homeButton.addEventListener('click', switchToHomeView);
+
+//event handlers
+function switchToFormView() {
```

Comment:
Maybe look into if you can apply toggle rather than add/remove in your refactoring. It'll help DRY up your code, possibly.

Reply:
Matt thought about toggle, but we didn't think it would be flexible since we want to navigate between three different places, instead of just two.
But we will talk through it in the morning.

Reply:
We still need to consider toggle, and will probably implement in Iteration 3.